app/assets/sass/patterns/_step-by-step-navigation.scss

```diff
@@ -0,0 +1,553 @@
+// Based on https://github.com/alphagov/govuk_publishing_components/blob/v9.3.6/app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss
+
+.app-step-nav-header {
+  position: relative;
+  padding: 10px;
+  background: govuk-colour("grey-4");
+  border-top: solid 1px $govuk-border-colour;
+  border-bottom: solid 1px $govuk-border-colour;
+  @include govuk-media-query($from: tablet) {
+    padding: 15px;
+  }
+}
+
+.app-step-nav-header__part-of {
+  @include govuk-font(16, $weight: bold)
+  display: block;
+  padding-bottom: 0.2em;
+}
+
+.app-step-nav-header__title {
+  @include govuk-font(24, $weight: bold)
+}
+
+.app-step-nav-related {
+  border-top: 2px solid $govuk-link-colour;
+  margin-bottom: 45px;
+}
+
+.app-step-nav-related__heading {
+  @include govuk-font(19, $weight: bold)
+  margin-top: 15px;
+  margin-bottom: 10px;
+}
+
+.app-step-nav-related__links {
+  @include govuk-font(16, $weight: bold)
+  margin: 0;
+  padding: 0;
+}
+
+.app-step-nav-related__pretitle {
+  display: block;
+  margin-bottom: 15px;
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: 5px;
+  }
+}
+
+.app-step-nav-related__links {
+  list-style: none;
+}
+
+.app-step-nav-related__link-item {
+  margin-top: 15px;
+  @include govuk-media-query($from: tablet) {
+    margin-top: 5px;
+  }
+}
+
+.app-step-nav {
+  @include govuk-font($size: 19);
+  margin-bottom: 30px;
+  &.app-step-nav--large {
+    @include govuk-media-query($from: tablet) {
+      margin-bottom: 60px;
```

Comment:
```suggestion
      margin-bottom: govuk-spacing(9);
```

Reply:
it's not quite 5xN

https://github.com/alphagov/govuk-frontend/blob/master/src/settings/_spacing.scss